### PR TITLE
[#172744149] Footer links are clickable even outside of the text

### PR DESCRIPTION
**Pull request by nykka**
Pivotal story reference: https://www.pivotaltracker.com/story/show/172744149

This PR:
- Changes footer's links display:inline-block to display:table to fix UX issue.
**THIS PR SHOULD BE MERGED TOGETHER WITH https://github.com/ombulabs/fastruby.io/pull/187**

### CODE REVIEW

File: vendor/assets/stylesheets/fastruby/styleguide/4-molecules/_footer.scss [hunk 1]
```diff
@@ -0,0 +1,91 @@
+footer {
+	background: $grey;
+	color: $white;
+	position: relative;
+	text-align: center;
+	padding-top: 5rem ;
+	padding-bottom: 10px;
+  overflow: hidden;
```
Comment (lubc) [inline vendor/assets/stylesheets/fastruby/styleguide/4-molecules/_footer.scss]: @nykka this line is not align with the others

```suggestion
	overflow: hidden;
```

File: vendor/assets/stylesheets/fastruby/styleguide/4-molecules/_footer.scss [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	}
+	#block-container {
+		text-align: center;
+		margin-top: 4rem;
+	}
+	#bloc1, #bloc2 {
+		display: inline-block;
+	}
+	.logo-footer {
+	  background-image: asset_url('isotype_vector.svg');
```
Comment (lubc) [inline vendor/assets/stylesheets/fastruby/styleguide/4-molecules/_footer.scss]: @nykka same here

File: vendor/assets/stylesheets/fastruby/styleguide/4-molecules/_footer.scss [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+		text-align: left;
+		padding-left: 40px;
+		background-size: auto 90%;
+		padding-top: 0px;
+		font-size: 1.8rem;
+		margin-bottom: 0;
+		a {
+			text-decoration: none;
+			top: -4px;
+    		position: relative;
```
Comment (lubc) [inline vendor/assets/stylesheets/fastruby/styleguide/4-molecules/_footer.scss]: @nykka and here

**Review by lubc — COMMENTED** [review]
@nykka Looks good. just some small comments regarding indentation

Also please fix the conflict so I can merge it. Thanks!

**Review by etagwerker — COMMENTED** [review]
@nykka could you review the conflicts here? thanks!

**Comment by lubc** [comment]
@nykka hey, please update this PR when you have some time. Thanks!

**Review by etagwerker — APPROVED** [review]
@nykka Looks good 👍

**Pull request merged**